server/startup/migrations/v192.js

```diff
@@ -0,0 +1,9 @@
+import { Migrations } from '../../../app/migrations';
+import { Settings } from '../../../app/models';
+
+Migrations.add({
+	version: 189,
```

Comment:
Wrong number and missing the import at `/server/startup/migrations/index.js`

Reply:
🤦  my bad. I forgot I had to do a hard reset and lost those changes.